plugins/jobtype/src/azkaban/jobtype/HadoopJavaJobRunnerMain.java

```diff
@@ -100,7 +100,7 @@ public void run() {
       Properties prop = new Properties();
```

Comment:
This is very easy and safe to do in Eclipse: Put your cursor over `prop`, press Alt + Shift + R (refactor), type in a new name, and press Enter.